src/Microsoft.SqlTools.ServiceLayer/SchemaCompare/Contracts/SchemaCompareOptions.cs

```diff
@@ -0,0 +1,207 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using Microsoft.SqlServer.Dac;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Text;
+
+namespace Microsoft.SqlTools.ServiceLayer.SchemaCompare.Contracts
+{
+    /// <summary>
+    /// Class to define schema compare and publish options. 
+    /// Keeping the order and defaults same as DacFx
+    /// The default values here should also match the default values in ADS UX
+    /// </summary>
+    public class SchemaCompareOptions
```

Comment:
Do we need the SchemaCompareOptions type? Could we reuse the existing options type defined for the publish/script tools service operations?

Reply:
@pensivebrian not sure which ones you are referring to. We did not pass any options so far to Schema Compare operations, adding this inside the existing Schema compare params only. Please let me know if I am missing something here...

Reply:
We had talked about exposing the deployment options in the dac deployment wizards, so it's probably not there - my bad. But if we do in the future, I think it makes sense to use the same type, so perhaps we should rename to this type to DeploymentOptions ?

Reply:
Sure I can do that (just wanted to double check to confirm - it wouldn't be confusing right?)

Reply:
Long term, are we going to have to deal with other schema compare options? For example, when we expose a checkbox to ignore a change - those options wouldn't make sense on a DeploymentOptions type. If that's the case, maybe it make more sense to make DeploymentOptions is own type which is a property of the SchemaCompareOptions type?

Reply:
Currently - SchemaComapreOption or DacDeployOptions both - have two types of things 1) the options to ignore/verify and so on and 2) excluded object types. But the individual diff or change excludes though checkboxes are not set through options - Schema Comparison Result has a separate API for those. 

So given the parity with DacDeployOptions - I am changing the name for now to DeploymentOptions as you suggested. We can revisit the name if it doesn't fit once we share the extension side of changes :)